src/qt/bitcoingui.cpp

```diff
@@ -74,6 +74,8 @@ const std::string BitcoinGUI::DEFAULT_UIPLATFORM =
 #endif
         ;
 
+/** Default wallet file name. Uses tilde to avoid name
```

Comment:
Calling this a file name may be deceptive. Maybe "display name for default wallet".